recipe/patches/0004-loosen-requirements.patch

```diff
@@ -1,16 +1,5 @@
-From 59100b60d2aeaa9110346995777b6ffc255e8999 Mon Sep 17 00:00:00 2001
-From: "Uwe L. Korn" <uwe.korn@quantco.com>
-Date: Wed, 8 Sep 2021 06:57:09 +0200
-Subject: [PATCH 4/5] loosen requirements
-
-* Set grpcio pin to the built grpc version
-* Remove unused clang requirement
```

Comment:
IMO we should try to keep patch attribution, but not a blocker.

Reply:
I mean, this patch has changed quite a bit from the original grpcio pinning change. I don't think the attribution is correct anymore.

Reply:
Again without this being a blocker, you could add a `Coauthored-By: ...` and update the description. Compare for example this [commit](https://github.com/conda-forge/python-feedstock/pull/535/commits/6a6ee5ceab60ded2b2aa8b6b21f20d5284edbbc3) (though, granted, that's a much more substantial patch).

Reply:
Ok i've updated the patch information.